### ISSUE TITLE
Error Flashing USB Bootloader

### ISSUE THREAD

**Issue by ndoolin** [phl0/MMDVM_HS_Dual_Hat#10]
Hi All!  I'm not sure if this is the right forum for this question, but if not, hopefully someone can point me in the right direction.

While attempting to run the following command.

make stlink-bl ././STM32F10X_Lib/utils/rpi32/st-flash write ./STM32F10X_Lib/utils/bootloader/generic_boot20_pc13.bin 0x8000000

It erases the flash and then it stops,  indicating.....

error flash_loader.c: unknown coreid, not sure what flash loader to use, aborting! coreid: 2ba01477, chipid: 410

I'm using an STLink V2 adapter hooked into the STLink Pins on the MMDVM.  Platform is a Raspberry Pi 4.  I ultimately want to use this board to hook directly into a PC and use with BlueDV.

Any Help is Appreciated.

Thanks!

-Nick

**Comment by phl0** [comment]
This is an issue of the firmware. This repo contains just the hardware. Sri

**Comment by ndoolin** [comment]
Thanks for the response.  I was able to get this to flash last night.  It seemed to be related to the version of st-flash not recognizing the chip on my board, I saw some posts suggesting a clone chip causing this issue.  I ended up downloading and compiling a newer version (1.71 ? )of stlink tools and I was able to successfully install the boot loader.  Still no luck yet getting it recognized properly by Windows or BlueDV yet…. but progress.  Thank You for your help!
